pages/index.vue

```diff
@@ -1,57 +1,104 @@
-const { data: speeches, pending: speechesPending } = useTracks({
-  contentType2: ["speech"],
-  size: 5,
-});
+let stopHandles: (() => void)[] = [];
+watch(
+  locale,
+  () => {
+    const { data, pending } = useDiscover({
+      // TODO: Find out how to set the age ...
+      // Compatibility is ensured in i18n.config.ts file
+      lang: locale.value as LanguageEnum,
+    });
+    stopHandles.forEach((el) => el());
 
-// Audiobooks
-const { data: audiobooks, pending: audiobooksPending } = useTracks({
-  contentType2: ["audiobook"],
-  size: 5,
-});
+    stopHandles = [
+      watch(
+        data,
+        (d) => {
+          discoveries.value = d;
+        },
+        { immediate: true }
+      ),
+      watch(
+        pending,
+        (p) => {
+          loading.value = p;
+        },
+        { immediate: true }
+      ),
+    ];
+  },
+  { immediate: true }
+);
 </script>
 
 <template>
   <div class="flex flex-col gap-16">
-    <ContentSection title="Playlists">
-      <div v-if="playlistsPending" class="flex h-64 gap-6">
-        <div
-          v-for="i in 4"
-          :key="i"
-          class="aspect-square h-full rounded-xl bg-background-2"
-        ></div>
-      </div>
-      <PlaylistCarousel
-        v-else-if="newestPlaylists"
-        :playlists="newestPlaylists"
-      />
-    </ContentSection>
-    <ContentSection :title="$t('nav.messages')" :link="{ name: 'messages' }">
-      <TrackList
-        :skeleton-count="5"
-        :show-skeleton="speechesPending"
-        :tracks="speeches"
-      >
-      </TrackList>
-    </ContentSection>
-    <ContentSection
-      :title="$t('nav.audiobooks')"
-      :link="{ name: 'audiobooks' }"
-    >
-      <TrackList
-        :skeleton-count="5"
-        :show-skeleton="audiobooksPending"
-        :tracks="audiobooks"
-      >
-      </TrackList>
-    </ContentSection>
+    <ul>
+      <template v-if="loading">
+        <li
+          v-for="index in 5"
+          :key="index"
+          class="my-6 h-11 w-full animate-pulse rounded-lg bg-background-2"
+        ></li>
+      </template>
+      <template v-else>
+        <li>
+          <template
+            v-for="item in discoveries"
+            :key="`${item.type}_${item.id}`"
+          >
+            <PageHeading v-if="item.type === 'section_header'" :level="3"
+              ><a v-if="typeof item.link === 'string'" :href="item.link"
+                >{{ item.title }}
+              </a>
+              <span v-else>{{ item.title }}</span></PageHeading
+            >
+            <NuxtLink
+              v-else-if="item.type === 'album'"
+              :to="{ name: 'album-id', params: { id: item.id } }"
+              style="display: inline-block; padding: 0 20px 20px 0"
+            >
+              <ProtectedImage
+                :src="item.cover || ''"
+                style="
+                  max-width: 214px;
+                  border: 1px solid rgba(129, 136, 143, 0.1);
+                  border-radius: 16px;
```

Comment:
Probably something like `class="max-w-[214px] border rounded-md"`

Reply:
Why use `rounded-md` here? Isn't the `-md` equal to `0.375rem`, which is `6px` for most systems..? And adding `border` here gives it a solid black border, not the color I got in the design.